xml/System.Diagnostics/ActivityLink.xml

```diff
@@ -195,8 +196,9 @@ Activity links can be used to represent batched operations where an activity was
       <Docs>
         <param name="left">The first value to compare.</param>
         <param name="right">The second value to compare.</param>
-        <summary>Returns a value that indicates whether two specified ActivityContext values are not equal.</summary>
-        <returns>true if left and right are not equal; otherwise, false.</returns>
+        <summary>Determines whether two specified <see langword="ActivityLink" /> values are not equal.</summary>
+        <returns>
+          <see langword="true" /> if <paramref name="left" /> and <paramref name="right" /> are note equal; otherwise, <see langword="false" />.</returns>
```

Comment:
```suggestion
          <see langword="true" /> if <paramref name="left" /> and <paramref name="right" /> are not equal; otherwise, <see langword="false" />.</returns>
```

Reply:
Sorry!